README.md

```diff
@@ -55,7 +56,7 @@ In [3]: %%prql
    ...:   aggregate [average unitPrice]
    ...: )
 In [4]: %%prql results <<
-   ...: from p = products.csv
+   ...: from p 
```

Comment:
Does this work? Is there a table called `p`? I realize it's that way above (which maybe I wrote!)..

Reply:
if 'p' is a table in the database yes, I reworked the %%prql magic docs to use a full database connection string, if using CSV I think we should push the pandas accessor.